### ISSUE TITLE
Convert onnx model to tensorflow with error "Process finished with exit code -1073741819 (0xC0000005)"

### ISSUE THREAD

**Issue by nicheng0019** [onnx/onnx-tensorflow#790]
I use pytorch to train a model and convert it to onnx format. I want to convert this onnx model to tensorflow format, the program 
crashes with error "Process finished with exit code -1073741819 (0xC0000005)".

My program is :

import onnx
from onnx_tf.backend import prepare

model = onnx.load('model.onnx')

tf_rep = prepare(model)

The crash happens at "tf_rep = prepare(model)".

onnx model file:  https://drive.google.com/file/d/1QBgBfVcWcqIBeqHrF8UbMlW4ATMH7O_p/view?usp=sharing

 - Python version:  3.6.5
 - ONNX version: 1.7.0
 - ONNX-TF version: 1.6.0
 - Tensorflow version:  2.3.0

model graph:
graph torch-jit-export (
  %input[FLOAT, input_dynamic_axes_1x1xinput_dynamic_axes_2xinput_dynamic_axes_3]
) initializers (
  %conv1.bias[FLOAT, 256]
  %conv1.weight[FLOAT, 256x1x3x3]
  %conv2.bias[FLOAT, 512]
  %conv2.weight[FLOAT, 512x256x3x3]
  %conv3.bias[FLOAT, 1024]
  %conv3.weight[FLOAT, 1024x512x3x3]
  %fc.bias[FLOAT, 10]
  %fc.weight[FLOAT, 10x1024]
) {
  %9 = Conv[dilations = [1, 1], group = 1, kernel_shape = [3, 3], pads = [0, 0, 0, 0], strides = [2, 2]](%input, %conv1.weight, %conv1.bias)
  %10 = Relu(%9)
  %11 = Conv[dilations = [1, 1], group = 1, kernel_shape = [3, 3], pads = [0, 0, 0, 0], strides = [2, 2]](%10, %conv2.weight, %conv2.bias)
  %12 = Relu(%11)
  %13 = Conv[dilations = [1, 1], group = 1, kernel_shape = [3, 3], pads = [0, 0, 0, 0], strides = [2, 2]](%12, %conv3.weight, %conv3.bias)
  %14 = Relu(%13)
  %15 = GlobalAveragePool(%14)
  %16 = Flatten[axis = 1](%15)
  %output = Gemm[alpha = 1, beta = 1, transB = 1](%16, %fc.weight, %fc.bias)
  return %output
}

Thanks!

**Comment by chinhuang007** [comment]
Can you provide access to the onnx file link? Thanks.

**Comment by nicheng0019** [comment]
> Can you provide access to the onnx file link? Thanks.

Hi, 
onnx model file: 
https://drive.google.com/file/d/1QBgBfVcWcqIBeqHrF8UbMlW4ATMH7O_p/view?usp=sharing

Can you access the file now?

**Comment by nicheng0019** [comment]
> Can you provide access to the onnx file link? Thanks.

@chinhuang007 
Hi,
onnx model file:
https://drive.google.com/file/d/1QBgBfVcWcqIBeqHrF8UbMlW4ATMH7O_p/view?usp=sharing

Can you access the file now?

**Comment by chinhuang007** [comment]
@nicheng0019 I can access the file now. And it works fine with prepare. I wonder you if you are using the latest code from the master branch and what platform you have?

**Comment by nicheng0019** [comment]
@chinhuang007 The ONNX-TF version is 1.6.0 and I run the code on windows. Do ONNX-TF support windows?

**Comment by chinhuang007** [comment]
ONNX-TF does not officially support Windows since we don't have CI for it. However, some people did run the converter on Windows before. Let's see if anyone can help verify your case.

**Comment by nicheng0019** [comment]
@chinhuang007 ok，thanks! I will run ONNX-TF on Linux
